test/run_test.py

```diff
@@ -340,8 +336,8 @@ def print_to_stderr(message):
     print(message, file=sys.stderr)
 
 
-def get_executable_command(options, allow_pytest, disable_coverage=False):
-    if options.coverage and not disable_coverage:
```

Comment:
Hmm, can we please keep the option in place, but simply no longer use it in the run_test.py?

Reply:
Updated